### PR TITLE
refactor:fixed multimeter UI

### LINKED ISSUE #1083: Text Views in multimeter knob are not perfectly positioned
**Actual Behaviour**

The text views of knob in multimeter are not perfectly positioned according to the knob

**Expected Behaviour**

The positioning should be made proper

**Steps to reproduce it**

Make changes in multimeter layout

**LogCat for the issue**

Not applicable

**Screenshots of the issue**

knob_bug

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by abhinavraj23**
Fixes #1083 

**Changes**:
Positioned the text-views according to the the knob

**Screenshot/s for the changes**: 
refact_mul

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**:
[multimeter_circle.apk.zip](https://github.com/fossasia/pslab-android/files/2121007/multimeter_circle.apk.zip)

### CODE REVIEW

**Comment by abhinavraj23** [comment]
@CloudyPadmal @Avjeet @harsh-2711 @Vikum94 Please review

File: build.gradle [hunk 1]
```diff
@@ -44,7 +44,7 @@ ext{
     commonLangVersion = '3.5'
     materialDialogsVersion = '0.9.6.0'
     navTabStripVersion = '1.0.4'
-    constraintLayoutVersion = '1.0.2'
+    constraintLayoutVersion = '1.1.0-beta3'
```
Comment (Vikum94) [inline build.gradle]: Is it good to use beta versions. They are still in testing.

Comment (harsh-2711) [reply build.gradle]: @abhinavraj23 I think the alpha version is available. You can use it.

File: app/src/main/res/layout/activity_multimeter.xml [hunk 2]
```diff
@@ -624,4 +366,169 @@
...
+        android:layout_height="wrap_content"
+        android:gravity="end"
+        android:text="@string/sen"
+        android:textStyle="bold"
+        app:layout_constraintCircle="@id/knobs"
+        app:layout_constraintCircleAngle="39"
+        app:layout_constraintCircleRadius="@dimen/multimeter_knobcircle_radius_1" />
+
+    <TextView
+        android:id="@+id/textView8"
```
Comment (CloudyPadmal) [inline app/src/main/res/layout/activity_multimeter.xml]: Rather than using `textviewxxx` let's use meaningful names.

**Comment by akarshan96** [comment]
@abhinavraj23 name the UI components as mentioned [here](https://github.com/fossasia/pslab-android#code-style).

**Comment by abhinavraj23** [comment]
@CloudyPadmal @Vikum94 Changes done :

**Comment by abhinavraj23** [comment]
Before:-
https://user-images.githubusercontent.com/32960257/41593433-a6eb4d8a-73dd-11e8-9c1a-3dffc58c9794.png

After:-
https://user-images.githubusercontent.com/32960257/41684239-85900392-74fa-11e8-963b-978cccf30782.png

**Comment by Vikum94** [comment]
@abhinavraj23 Are you hoping to close this PR, because there is another PR mentioned by @CloudyPadmal

**Comment by abhinavraj23** [comment]
Yes okay I will close it @Vikum94

**Pull request closed without merging**